### ISSUE TITLE
SSI - server side includes ?

### ISSUE THREAD

**Issue by centminmod** [h2o/h2o#243]
Any plans for h2o to include SSI support http://nginx.org/en/docs/http/ngx_http_ssi_module.html ?

**Comment by kazuho** [comment]
I would rather go towards supporting [ESI](http://en.wikipedia.org/wiki/Edge_Side_Includes).

Technically speaking, we could support SSI as part of / after adding support for CGI.

**Comment by centminmod** [comment]
yeah I use nginx SSI extensively for plain static html sites to add common header and footer templates via SSI so would love to have similar functionality in h2o :)

**Comment by kazuho** [comment]
It would be interesting to implement SSI using #1217.
